packages/dev/webpack.config.js

```diff
@@ -8,7 +8,7 @@ const path = require('path')
 const isDevelopment = process.env.NODE_ENV !== 'production'
 module.exports = {
   entry: './src/index.tsx',
-  devtool: 'source-map',
+  devtool: isDevelopment ? 'eval-source-map' : 'source-map',
```

Comment:
🤔 I remember having no issues with debugging Unovis from the dev app. Was the error in the console also leading to the obfuscated bundle? 

Also the `source-map` option generally produces better results that 'eval-source-map' and is more computationally expensive to calculate. Do you know why the `eval-source-map` works here better here?

Reply:
This change changes the error message from pointing to `bundle.js` to pointing to something from `node_modules`. That's arguably an improvement, but actually it's not that much better, since we still are not getting the line numbers for the original error. Happy not to merge this, maybe it's just a weird case I'm dealing with. Thanks for reviewing!

Reply:
But overall, the source maps don't seem to be working.

For example, when I add this change:

<img width="806" alt="image" src="https://github.com/user-attachments/assets/cdd9d3bf-9106-45c9-92a6-a9a2c6d6b5ee">

I get this error:

<img width="891" alt="image" src="https://github.com/user-attachments/assets/14e550e8-c0e0-411a-8ba9-df6db1f6a6a4">

The source maps don't seem to be doing their job. Instead of `bundle.js` it should resolve to the specific line number and source file, but it does not.

Reply:
After changing it to `eval-source-map`, the line numbers DO resolve correctly.

<img width="919" alt="image" src="https://github.com/user-attachments/assets/2e9f8cfd-2f6b-4310-822e-2cd578da17b8">